app/services/to_cocina/work/types_mapper.rb

```diff
@@ -68,7 +68,7 @@ def genres
         return [] if work_type == 'Other'
 
         # Uniq avoids adding dupe genres when same genre in from type and subtype.
```

Comment:
Not part of your code change, but typo?
```suggestion
        # Uniq avoids adding dupe genres when same genre is from type and subtype.
```